xla/pbgenerator.py

```diff
@@ -0,0 +1,69 @@
+   0.415734806, -0.097545161, -0.490392640, -0.277785117, 0.277785117, 0.490392640, 0.097545161, -0.415734806,
+   0.353553391, -0.353553391, -0.353553391, 0.353553391, 0.353553391, -0.353553391, -0.353553391, 0.353553391,
+   0.277785117, -0.490392640, 0.097545161, 0.415734806, -0.415734806, -0.097545161, 0.490392640, -0.277785117,
+   0.191341716, -0.461939766, 0.461939766, -0.191341716, -0.191341716, 0.461939766, -0.461939766, 0.191341716,
+   0.097545161, -0.277785117, 0.415734806, -0.490392640, 0.490392640, -0.415734806, 0.277785117, -0.097545161], shape=[8, 8], dtype=tf.float32)
+
+dct_transp = tf.compat.v1.constant([0.353553391, 0.490392640, 0.461939766, 0.415734806, 0.353553391, 0.277785117, 0.191341716, 0.097545161,
+      0.353553391, 0.415734806, 0.191341716, -0.097545161, -0.353553391, -0.490392640, -0.461939766, -0.277785117,
+      0.353553391, 0.277785117, -0.191341716, -0.490392640, -0.353553391, 0.097545161, 0.461939766, 0.415734806,
+      0.353553391, 0.097545161, -0.461939766, -0.277785117, 0.353553391, 0.415734806, -0.191341716, -0.490392640,
+      0.353553391, -0.097545161, -0.461939766, 0.277785117, 0.353553391, -0.415734806, -0.191341716, 0.490392640,
+      0.353553391, -0.277785117, -0.191341716, 0.490392640, -0.353553391, -0.097545161, 0.461939766, -0.415734806,
+      0.353553391, -0.415734806, 0.191341716, 0.097545161, -0.353553391, 0.490392640, -0.461939766, 0.277785117,
+      0.353553391, -0.490392640, 0.461939766, -0.415734806, 0.353553391, -0.277785117, 0.191341716, -0.097545161], dtype=tf.float32, shape=[8, 8])
+
+flow_coeff = tf.compat.v1.constant([8.000000,  11.096321, 10.452571, 9.406975,  8.000002,  6.285559,  4.329570,  2.207195,
+                                      11.096320, 15.391046, 14.498035, 13.047892, 11.096315, 8.718322,  6.005284,  3.061468,
+                                      10.452501, 14.498041, 13.656854, 12.290866, 10.452503, 8.212481,  5.656853,  2.883838,
+                                      9.407003,  13.047894, 12.290842, 11.061483, 9.407002,  7.391037,  5.091034,  2.595389,
+                                      8.000001,  11.096319, 10.452504, 9.407007,  8.000000,  6.285560,  4.329569,  2.207195,
+                                      6.285562,  8.718323,  8.212480,  7.391035,  6.285561,  4.938532,  3.401721,  1.734182,
+                                      4.329566,  6.005280,  5.656854,  5.091043,  4.329567,  3.401721,  2.343146,  1.194525,
+                                      2.207195,  3.061468,  2.883839,  2.595387,  2.207199,  1.734182,  1.194525,  0.608964], dtype=tf.float32, shape=[8, 8])
+
+# This placeholder is where the inference in C++ starts. Therefore the name
+# set here will be used later as 'input_node_name'
+x   = tf.compat.v1.placeholder(dtype=tf.float32, shape=[8, 8], name="x")
+y   = tf.compat.v1.multiply(tf.compat.v1.matmul(tf.compat.v1.matmul(dct_matrix, x), dct_transp), flow_coeff, name="y")
```

Comment:
Define intermediate value for readability.

dct_x_dct_transp = tf.compat.v1.matmul(tf.compat.v1.matmul(dct_matrix, x), dct_transp)

(Again, watch 80 character limit, seems to be exceed all over this file)